apps/platform-app/src/components/hunter-landing/HunterHeader/HunterHeader.tsx

```diff
@@ -1,7 +1,67 @@
+'use client'
+import { LocaleSwitcher } from '@/app/[locale]/(profile)/(components)/LocaleSwitcher/LocaleSwitcher'
+import { I18nNamespaces } from '@/i18n/request'
+import { AppRoutes } from '@/utils/routes'
+import { Button } from '@gdh/ui-system'
+import { useTranslations } from 'next-intl'
+import { useRouter } from 'next/navigation'
+import { useState } from 'react'
+import { LogoLight } from '../UI/LogoLight/LogoLight'
 import styles from './HunterHeader.module.scss'
 
-type Props = {}
+export const HunterHeader = () => {
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false)
+  const [menuVisible, setMenuVisible] = useState(false)
+  const t = useTranslations(I18nNamespaces.HunterHeader)
+  const router = useRouter()
 
-export const HunterHeader = (props: Props) => {
-  return <div className={styles.wrapper}> HunterHeader</div>
+  const handleAddJob = () => {
+    router.push(AppRoutes.postJob)
+  }
+
+  const toggleMobileMenu = () => {
+    if (isMobileMenuOpen) {
+      setMenuVisible(false)
+      setTimeout(() => {
+        setIsMobileMenuOpen(false)
+      }, 300)
+    } else {
+      setIsMobileMenuOpen(true)
+      setTimeout(() => {
+        setMenuVisible(true)
+      }, 10)
+    }
+  }
+
+  return (
+    <section id="HunterHeader" className={styles.wrapper}>
+      <LogoLight />
+      <div className={styles.menuDesktop}>
+        <LocaleSwitcher variant="hunterLanding" />
+
+        <Button variant="allpurple" onClick={handleAddJob}>
+          {t('addJob')}
+        </Button>
+      </div>
+      <div className={styles.menuMobile}>
+        <button className={styles.mobilebutton} onClick={toggleMobileMenu}>
+          {isMobileMenuOpen ? 'X' : 'Menu'}
```

Comment:
You replace "Menu" with "X" but don't change the button size - Consider using "Close" instead of "X"